File: Backend/src/main/java/sr57/ftn/reddit/project/model/dto/postDTOs/AddPostDTO.java
package sr57.ftn.reddit.project.model.dto.postDTOs;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;
import org.springframework.lang.Nullable;

import java.io.Serializable;

@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
public class AddPostDTO implements Serializable {
    private String title;
    private String text;
    @Nullable
    private int flair_id;
}

File: NewFront/src/app/dto/addPostDTO.ts
export class AddPostDTO {
    title: string = "";
    text: string = "";
    flair_id: number = 0;

    AddPostDTO(title: string, text: string, flair_id: number) {
        this.title = title;
        this.text = text;
        this.flair_id = flair_id;
    }
}